src/Sarif/NotYetAutoGenerated/SarifRewritingVisitor.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// Rewriting visitor for the Sarif object model.
     /// </summary>
     [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.58.0.0")]
-    public abstract class SarifRewritingVisitor
+    public abstract partial class SarifRewritingVisitor
```

Comment:
>SarifRewritingVisitor [](start = 34, length = 21)

Created this class to help minimize constant manual updates for generated file. There are still at least three manual changes required for emitted file.

Reply:
Add a code comment to that effect.

---
In reply to: [244650147](https://github.com/Microsoft/sarif-sdk/pull/1194#discussion_r244650147) [](ancestors = 244650147)